.asyncapi-tool.yaml

```diff
@@ -0,0 +1,13 @@
+title: AsyncAPI Server API
+description: Server API providing official AsyncAPI tools.
+links:
+  websiteUrl: https://api.asyncapi.com/v1
+  docsUrl: https://api.asyncapi.com/v1/docs
+filters:
+  language: JavaScript
+  technology:
+    - Express.js
+    - TypeScript
+  categories:
```

Comment:
imho API is enough. 

category is a grouping parameter, `server` group will not really bring much value for person looking for tools

Reply:
As I see, we don't have `api` category. Should I add this, or someone else (like you)?

Reply:
you can add it, do it for `editor` too